populus/utils/epoch.py

```diff
@@ -0,0 +1,5 @@
+import time
+
+
+def epoc_time_now():
+    return time.time()
```

Comment:
We might want to consider truncating this to the integer representation since `time.time()` returns a float.